docs/haskell.rst

```diff
@@ -25,9 +25,26 @@ In this tutorial you'll learn how to:
 Before you begin
 ----------------
 
-To prepare for the tutorial, first `install Bazel`_ if you don't have
-it installed already. Then, retrieve the ``rules_haskell`` GitHub
-repository::
+On a Unix system you will need the following tools installed.
+
+* ``gcc``
+* ``libffi``
+* ``libgmp``
+* ``libtinfo5``
+* ``make``
+* ``python3`` (``/usr/bin/env python`` needs to be available)
```

Comment:
```suggestion
* ``python3`` (``python`` needs to be available in ``$PATH`` as well, depending on your distro, this might require ``python2`` too)
```